### PR TITLE
Feature/whitelabel part 2

### PR DESCRIPTION
**Pull request by servusrene**
## What does this PR do?

continuation of the whitelabel

**Environment**: Staging(main branch) / Production

## Type of change
Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

Change branding environment variables to a different value as default:

```
NEXT_PUBLIC_APP_NAME="acme.com"
NEXT_PUBLIC_SUPPORT_MAIL_ADDRESS="support@acme.com"
NEXT_PUBLIC_COMPANY_NAME="ACME inc."
```

### CODE REVIEW

**Comment by rene-mueller** [comment]
@PeerRich @zomars i have continued what @OMaxOe has started

**Comment by rene-mueller** [comment]
@zomars can you check the failing steps? I can and do not find the error

**Comment by PeerRich** [comment]
all of the app store has to stay Cal branded / Cal emails because we ultimately provide support for reporting & updating apps, not the company white-labeling

**Comment by PeerRich** [comment]
and apps/web/components/ui/PoweredByCal.tsx should stay cal.com, too for now

File: apps/web/components/ui/PoweredByCal.tsx [hunk 1]
```diff
@@ -12,22 +12,22 @@ const PoweredByCal = () => {
...
+                alt="Cal.com Logo"
+              />
+              <img
+                className="relativ -mt-px hidden h-[10px] w-auto dark:inline"
+                src="/cal-logo-word-dark.svg"
+                alt="Cal.com Logo"
+              />
+            </>
+          ) : (
+            <span className="font-semibold">{APP_NAME}</span>
```
Comment (PeerRich) [inline apps/web/components/ui/PoweredByCal.tsx:29]: for now, we will keep powered by Cal.com

**Review by PeerRich — DISMISSED** [review]
blocking. a lot of these whitelabel actions will require a /ee/ license

File: apps/web/components/ui/PoweredByCal.tsx (hunk 1 above)
Comment (rene-mueller) [reply apps/web/components/ui/PoweredByCal.tsx:29]: is that an /ee/ feature too?

**Comment by rene-mueller** [comment]
> all of the app store has to stay Cal branded / Cal emails because we ultimately provide support for reporting & updating apps, not the company white-labeling

i understand then should the following string changed too:

```
  "every_app_published": "Every app published on the {{appName}} App Store is open source and thoroughly tested via peer reviews. Nevertheless, {{companyName}} does not endorse or certify these apps unless they are published by {{appName}}. If you encounter inappropriate content or behaviour please report it.",
```

File: apps/web/components/ui/PoweredByCal.tsx (hunk 1 above)
Comment (PeerRich) [reply apps/web/components/ui/PoweredByCal.tsx:29]: yes

**Comment by rene-mueller** [comment]
@PeerRich I'm done :)

**Comment by rene-mueller** [comment]
@zomars i'm not sure about the unit test error 
grafik
 
in the code the marked line dosn't exist ...

File: apps/web/components/ui/PoweredByCal.tsx (hunk 1 above)
Comment (ddm231) [reply apps/web/components/ui/PoweredByCal.tsx]: is there a chance to get a licence for white labeling in future?

Comment (rene-mueller) [reply apps/web/components/ui/PoweredByCal.tsx]: @ddm231 it will be an /ee/ license required: https://cal.com/platform

**Review by zomars — APPROVED** [review]
Thank you for your contribution 🙏 @rene-mueller

**Pull request merged**